spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc

```diff
@@ -944,6 +944,10 @@ content into your application; rather pick only the properties that you need.
 	spring.kafka.consumer.key-deserializer= # Deserializer class for keys.
 	spring.kafka.consumer.max-poll-records= # Maximum number of records returned in a single call to poll().
 	spring.kafka.consumer.value-deserializer= # Deserializer class for values.
+	spring.kafka.jaas.control-flag=REQUIRED # AppConfigurationEntry.LoginModuleControlFlag value.
+	spring.kafka.jaas.enabled= # Enable JAAS configuration.
```

Comment:
The documentation here _must_ be a copy/paste of the javadoc. This version is better than what you have in code.